src/platform/macos.m

```diff
@@ -270,4 +271,36 @@ uint8 platform_get_locale_measurement_format()
 	}
 }
 
+@interface RCT2MacUrlHandle : NSObject
+@end
+
+@implementation RCT2MacUrlHandle
+
+- (void)handleGetUrlEvent:(NSAppleEventDescriptor *)event withReplyEvent:(NSAppleEventDescriptor *)replyEvent
+{
+	const char *url = [[event paramDescriptorForKeyword:keyDirectObject] stringValue].UTF8String;
+	
+	SDL_Event sdl_event;
+	sdl_event.user.type = gOpenRCT2EventType;
+	sdl_event.user.code = OPENRCT2_EVENT_OPEN_SCHEME;
```

Comment:
It looks like it would be used to handle all URI events though which is why I asked.